SW_Flow_lib.c

```diff
@@ -2008,27 +2027,33 @@ Equations based on Eitzinger, Parton, and Hartman 2000. @cite Eitzinger2000, Par
 @param sTconst Constant soil temperature (&deg;C).
 @param deltaX Distance between profile points (default is 15 cm from Parton's equation @cite Parton1984).
 @param theMaxDepth Lower bound of the equation (default is 180 cm from Parton's equation @cite Parton1984).
-@param nRgr Number of regressions (1 extra value is needed for the sTempR and oldsTempR for the last layer.
+@param nRgr Number of regressions (1 extra value is needed for the avgLyrTempR and oldavgLyrTempR for the last layer.
 @param snow Snow-water-equivalent of the area (cm).
 @param *ptr_stError Boolean indicating whether there was an error.
+@param max_air_temp Maximum air temperature of Today
+@param min_air_temp Minimum air temperature of Today
+@param H_gt Daily global (tilted) irradiation [MJ / m2]

```

Comment:
Great that these new function parameters are documented, please also document the other new ones